ops/test/metabase.tf

```diff
@@ -14,4 +14,30 @@ module "metabase" {
 
   postgres_server_name = data.terraform_remote_state.persistent_test.outputs.postgres_server_name
   postgres_url         = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.metabase_db_uri.id})"
+}
+
+module "metabase-readonly" {
+  source = "../services/app_service/metabase"
+  name   = "${local.project}-${local.name}-${local.env}-metabase-readonly"
+  env    = local.env
+
+  app_settings = {
```

Comment:
This part is going to fail a `terraform plan`. If you look at `ops/services/app_service/metabase/_var.tf`, you'll see a bunch of variables that the metabase module expects/accepts. `app_settings` isn't one of them, since it's an argument to the metabase resource itself rather than the module - this will throw an unexpected variable argument.

We want to use the `app_settings_overrides` variable we added above here instead.

Reply:
Done, thanks for the detailed explanation!